osv2/mget.go

```diff
@@ -0,0 +1,146 @@
+package osv2
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"net/http"
+
+	"github.com/opensearch-project/opensearch-go/v2"
+	"github.com/opensearch-project/opensearch-go/v2/opensearchapi"
+
+	"github.com/CrowdStrike/opensearchtools"
+)
+
+// MGetRequest is a marshalable form of [opensearchtools.MGetRequest] specific to the opensearchapi.MgetRequest in OpenSearch v2.
```

Comment:
do we want to link the 2.x documentation for multi-get?

Reply:
I can't seem to get a link to 2.x - I can either do a specific version or `latest`. I could put in 2.5, but then that would be out of date when they add 2.6. I could link to `latest`, but then that will be wrong when they add version 3.

Reply:
I agree with Jeff on this one.  There's no URL to the docs for v2.x so we shouldn't try to DIY it.

Reply:
fair enough